src/components/ResizableContainer.tsx

```diff
@@ -0,0 +1,120 @@
+
```

Comment:
This is a good start, thanks for filing an issue for this feature. Can you please look at the mobile view once? We don't really want the resize option in mobile view. We are not optimizing this app for mobiles as of now.